### PR TITLE
av98: Add a formula for AV-98 v1.0.1

### PR DESCRIPTION
**Pull request by sevan**
That gives us a Gemini client via Python 3.

Ideally needs https://github.com/mistydemeo/tigerbrew/pull/805,  #823

### CODE REVIEW

File: Library/Formula/av98.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class Av98 < Formula
+  desc "AV-98 is an experimental client for the Gemini protocol"
+  homepage "https://tildegit.org/solderpunk/AV-98"
+  url "https://tildegit.org/solderpunk/AV-98/archive/v1.0.1.tar.gz"
```
Comment (mistydemeo) [inline Library/Formula/av98.rb:4]: Hmm, this is serving a 500. Is there a good mirror somewhere?

Comment (sevan) [reply Library/Formula/av98.rb:4]: Thanks for the heads up, it seems like the Tilde services have been experiencing issues in the last week, similar problem with another of their services. I'll see if I can find a mirror and get someone to poke at it.

Comment (mistydemeo) [reply Library/Formula/av98.rb:4]: Looks like it's still down, unfortunately. :/

**Comment by mistydemeo** [comment]
Nice, I see you found a mirror! I'll give this a try.

**Comment by sevan** [comment]
Cool, thanks :)

**Review by mistydemeo — APPROVED** [review]
I see it's pure Python with no build dependencies, so I won't bother bottling.

**Pull request merged**